src/parser/monk/mistweaver/modules/spells/EnvelopingMists.js

```diff
@@ -24,15 +23,32 @@ class EnvelopingMists extends Analyzer {
     combatants: Combatants,
   };
 
+  constructor(...args) {
+    super(...args);
+    if (this.selectedCombatant.hasTalent(SPELLS.MIST_WRAP_TALENT.id)) {
+      this.EVM_HEALING_INCREASE=.4;
```

Comment:
2 Style issues here:

1. Use camel case for the variable name, eg. `evmHealingIncrease`
2. Space the statement out, eg `this.evmHealingIncrease = .4;

Reply:
1. I was just reusing the old variable name but I'll change it
2. I'm going to switch it to a ternary but will make the spacing nice.